### PR TITLE
Create slyness.json

### PR DESCRIPTION
**Pull request by slyness6**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/slyness.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        "description": "Personal Website for Slyness",
+        "username": "slyness6",
+        "email": "slynesss@proton.me"
+        "discord": "lbt6"
+    },
+    "record": {
+    "NS": [
+	"jake.ns.cloudflare.com",
+	"stephane.ns.cloudflare.com"
+    ]
```
Comment (CuteDog5695) [inline domains/slyness.json:12]: Can you give a reason as to why you need NS records?

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/slyness.json (hunk 1 above)
Comment (slyness6) [reply domains/slyness.json:12]: > Can you give a reason as to why you need NS records?
bc ill be using it as a personal website and i need the ns records

Comment (slyness6) [reply domains/slyness.json:12]: bc i need to link it wit cloudflare

Comment (CuteDog5695) [reply domains/slyness.json:12]: Not a valid reason.

**Review by iostpa — CHANGES_REQUESTED** [review]
Reason is invalid. You can replace the records with other records which aren't NS records.

**Pull request closed without merging**